docs/updating-components.md

```diff
@@ -0,0 +1,123 @@
+# How to update components?
```

Comment:
Same here, maybe we can create `docs/development` sub-directory and put it there?